pySDC/projects/Second_orderSDC/dampedharmonic_oscillator_run_stab_interval.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from pySDC.projects.Second_orderSDC.penningtrap_Simulation import Stability_implementation
+from pySDC.projects.Second_orderSDC.dampedharmonic_oscillator_run_stability import dampedharmonic_oscillator_params
+from tabulate import tabulate
+
+
+def stab_interval(
+    num_nodes_list, max_iter_list, save_file=False, filename='./data/stab_interval.txt'
+):  # pragma: no cover
+    # Initialize simulation parameters based on the damped harmonic oscillator
+    description = dampedharmonic_oscillator_params()
+
+    # Initialize data as a list of rows
+    data = []
+
+    # Loop through different numbers of nodes, maximum iterations, and quadrature types
```

Comment:
Above, you forgot to mention quadrature types, but looped through them. Here you mention them, but don't loop through different quadrature nodes. Best remove the comment.